usagov/vote-gov-drupal
# Vote-443: Uninstall and remove gulpstyle lint to fix Postcss dependabot warning

## Jira ticket

[Vote-443](https://cm-jira.usa.gov/browse/VOTE-443)

## Description

This PR will update this [Dependabot Alert](https://github.com/usagov/vote-gov-drupal/security/dependabot/17). The work was done by uninstalling `gulpstyle-lint` that was being flagged.

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. Pull down branch 
2. cd into `web/themes/custom/vote_gov` and run `npm audit` and check that there is no alert for `Postcss`

## Checklist for the Developer




## Checklist for the Peer Reviewers
